test/integration/api.bats

```diff
@@ -450,9 +488,32 @@ function teardown() {
 	[[ "${lines[1]}" ==  *"Up"* ]]
 }
 
-# FIXME
 @test "docker stats" {
-	skip
+	TEMP_FILE=$(mktemp)
+	start_docker 3
+	swarm_manage
+	# stats running container 
+	run docker_swarm run -d --name test_container busybox sleep 50
+	[ "$status" -eq 0 ]
+
+	# make sure container is up
+	run docker_swarm ps -l
+	[ "${#lines[@]}" -eq 2 ]
+	[[ "${lines[1]}" == *"test_container"* ]]
+	[[ "${lines[1]}" == *"Up"* ]]
+
+	# storage the real time output of stats in TEMP_FILE
+	docker_swarm stats test_container > $TEMP_FILE &
+
+	# make sure TEMP_FILE exists and is not empty
+	sleep 3
```

Comment:
No sleeps like this please - could you do a loop that retries and sleeps a little bit between tries? Like in `wait_until_reachable`